### DISCUSSION TITLE
Maintenance in progress?

### DISCUSSION THREAD

**Discussion by aliasatp** [opengisch/QField#3753] [Bugs]
Hi, I have sync errors/fails (to and from qfieldcloud) since yesterday. Is there any maintanance in progress?

Start QGIS Application
08:20:39.622 QGSSTDERR INFO     Starting QGIS app version 32601 (exported)...
08:20:39.628 QGSSTDERR WARNING  QStandardPaths: XDG_RUNTIME_DIR not set, defaulting to '/tmp/runtime-root'
Download Project Directory
08:22:52.642 QGSSTDERR WARNING QFSFileEngine::open: No file name specified
08:22:52.642 QGSSTDERR INFO Stopping QGIS app…

**Comment by suricactus** [comment]
Hi @aliasatp. Thanks for posting this, there is a connectivity issue we are investigating now. Follow the thread here updates here https://qfield.checkly-dashboards.com/incidents/034186f9-3f86-4770-8a27-abb44b1d5848

**Reply by suricactus** [reply]
Closed for two days already, all looks good.
